lib/posix/options/posix_internal.h

```diff
@@ -29,6 +29,7 @@ struct posix_thread_attr {
 	uint16_t guardsize : CONFIG_POSIX_PTHREAD_ATTR_GUARDSIZE_BITS;
 	int8_t priority;
 	uint8_t schedpolicy: 2;
+	bool contentionscope;
```

Comment:
@cfriedt did you mean
```suggestion
	bool contentionscope: 1;
```
?

Reply:
if we want to maintain it with only one bit size seems right.